tests/queries/0_stateless/02994_libarchive_compression.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash
# Tags: no-fasttest
```

fast tests just compile binary without any libs so we simply disable tests for it which do need some of the libs